modules/orchestrator/proc-install-rhdh-orchestrator-airgapped-env-using-operator.adoc

```diff
@@ -0,0 +1,106 @@
++
+[source,subs="+attributes,+quotes"]
+----
+curl -sSLO https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/release-{product-version}/.rhdh/scripts/prepare-restricted-environment.sh
+----
+
+. Create an `ImageSetConfiguration` file for `oc-mirror`. Because `oc-mirror` does not automatically mirror all images required by the Serverless Logic Operator, you must use an `ImageSetConfiguration` file to include them as shown in the following example:
++
+[source,subs="+attributes,+quotes"]
+----
+apiVersion: mirror.openshift.io/v2alpha1
+kind: ImageSetConfiguration
+mirror:
+  additionalimages:
+  - name: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-jobs-service-ephemeral-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-data-index-postgresql-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-data-index-ephemeral-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-db-migrator-tool-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:{rhoserverless-version}.0
+  - name: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:{rhoserverless-version}.0
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.19
+      packages:
+      - name: logic-operator-rhel8
+        channels:
+        - name: alpha
+          minVersion: {rhoserverless-version}.0
+          maxVersion: {rhoserverless-version}.0
+      - name: serverless-operator
+        channels:
+        - name: stable
+          minVersion: {rhoserverless-version}.0
+          maxVersion: {rhoserverless-version}.1
+----
++
+Alternatively, you can use the podman commands to find the missing images and add them to the `additionalimages` list if the versions change:
++
+[source,subs="+attributes,+quotes"]
+----
+IMG=registry.redhat.io/openshift-serverless-1/logic-operator-bundle:{rhoserverless-version}.0-8
+mkdir local-manifests-osl
+podman create --name temp-container "$IMG" -c "cat /manifests/logic-operator-rhel8.clusterserviceversion.yaml"
+podman cp temp-container:/manifests ./local-manifests-osl
+podman rm temp-container
+yq -r '.data."controllers_cfg.yaml" | from_yaml | .. | select(tag == "!!str") | select(test("^.*\\/.*:.*$"))' ./local-manifests-osl/manifests/logic-operator-rhel8-controllers-config_v1_configmap.yaml
+----
+
+. Mirror the images in the `ImageSetConfiguration.yaml` file by running the `oc-mirror` command. For example:
++
+[source,subs="+attributes,+quotes"]
+----
+oc-mirror --config=imagesetconfiguration.yaml docker://<registry URL:port> --workspace file://<workspace folder> --authfile /path/to/authfile  --v2
+cd <workspace folder>/working-dir/cluster-resources/
+oc apply -f .
+----
```

Comment:
Ideally (and similar to how https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.7/html/installing_red_hat_developer_hub_in_an_air-gapped_environment/index is structured), we should split this into 2 different disconnected use cases:
- Partially disconnected use case, where the user can mirror the images directly from their laptop to the mirror registry in the disconnected env: this is covered by the `oc-mirror` command above
- Fully disconnected use case, where the user first needs to mirror the images to disk, then transfer the result into their physically disconnected env, then mirror from disk to their mirror registry.